sigstore/_cli.py

```diff
@@ -122,12 +162,28 @@ def _sign(
             ctfe_pem=ctfe_pem,
         )
 
-        click.echo("Using ephemeral certificate:")
```

Comment:
Flagging for review: I wasn't sure how best to preserve this presentation, now that `sig_output` and `cert_output` might be non-terminals. I could do that check, if these precise output lines are desirable in terms of compatibility with `cosign`.

Reply:
Here's what `cosign` does:

```
$ COSIGN_EXPERIMENTAL=1 cosign sign-blob hi.txt --output-signature out.sig --output-certificate out.crt
Using payload from: hi.txt
Generating ephemeral keys...
Retrieving signed certificate...
Successfully verified SCT...
using ephemeral certificate:
-----BEGIN CERTIFICATE-----
MIICMDCCAbWgAwIBAgITRoPTsLYp1eUpjsDZCs8h889GjjAKBggqhkjOPQQDAzAq
MRUwEwYDVQQKEwxzaWdzdG9yZS5kZXYxETAPBgNVBAMTCHNpZ3N0b3JlMB4XDTIy
MDUzMTIwNDY1M1oXDTIyMDUzMTIwNTY1MlowADBZMBMGByqGSM49AgEGCCqGSM49
AwEHA0IABOAJOtj9OZljXZ5Kf6CWvIaps73kqAfP2YtBGOKC7lsUFw+JnynYGuNr
EhNgZ2J7QtRsA6swJnwr8r4WQRAzCnKjgeMwgeAwDgYDVR0PAQH/BAQDAgeAMBMG
A1UdJQQMMAoGCCsGAQUFBwMDMAwGA1UdEwEB/wQCMAAwHQYDVR0OBBYEFFUs5VG3
O4TNa7EBDcx0N+KwHQo4MB8GA1UdIwQYMBaAFFjAHl+RRaVmqXrMkKGTItAqxcX6
MEAGA1UdEQEB/wQ2MDSBMjkxOTQzNjE1ODIzNi1jb21wdXRlQGRldmVsb3Blci5n
c2VydmljZWFjY291bnQuY29tMCkGCisGAQQBg78wAQEEG2h0dHBzOi8vYWNjb3Vu
dHMuZ29vZ2xlLmNvbTAKBggqhkjOPQQDAwNpADBmAjEAg7e1BPKz7oxJ7E4zCGCw
3Pz9Z1ZzCwbK1Tw2+v9iUZNyc4hpFs9hq0mLUV7s9oC2AjEAz4RWI5Yx0WQS/2BO
vsjHw2XUlTYLF3MJU/E2X7QbjP7KmDrCVISlYQ006jZGI7pe
-----END CERTIFICATE-----

tlog entry created with index: 2515095
Signature wrote in the file out.sig
Certificate wrote in the file out.crt
```

IMO, we should do the same.

Reply:
...although I'm not so sure about the grammar 'signature wrote in the file'...

Reply:
Pushed [0a79380](https://github.com/sigstore/sigstore-python/pull/101/commits/0a7938029c98104e4f4220d7935c3c643d8421b8) to match the `cosign` output.